### PR TITLE
UI Improvements for multiple tickets

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look. Changes include:
- Improving teams listing on the root page.
- Improving date separator and new messages separator for comment hover.
- Improving manage members and add members modal.
- Globally limiting max-height for modal-body if the window height exceeds 1200px.

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: web/react/components/more_direct_channels.jsx [hunk 1]
```diff
@@ -206,7 +206,7 @@ export default class MoreDirectChannels extends React.Component {
 
         return (
             <Modal
-                className='modal-direct-channels'
+                className='more-modal'
```
Comment (hmhealey) [inline web/react/components/more_direct_channels.jsx]: This should actually be dialogClassName when we're using React-Bootstrap Modals. I just found that out while doing all the stuff with UserSettings

**Comment by hmhealey** [comment]
Other than that, it looks good

**Comment by asaadmahmoodspin** [comment]
I'll resubmit a PR for this, messed it up trying to rebase.

**Comment by hmhealey** [comment]
Looks good

**Pull request merged**